### PR TITLE
Optimize info given in Request Panel

### LINKED ISSUE #5: 02. Request Panel
The overview has too many texts, we can reduce it, a developer loves less reading more bullets/sentences/images/code

We can show the list of inner components in `REQUEST PANEL`
```
1. Body
     - Quick method selection
     - Quick body type selection
     - plus icon for multiple body creation

2.  Auths
      - Quick auth type selection
3. Headers
4. Params
5. Scripts
    - pre
    - post
    - test script
6. Config
7. Code snippets preview 
```
image

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Fixes #5

### CODE REVIEW

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e7235ad4c23df021fae5fa3--firedoc.netlify.com

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e7621edfab06264b0dfb928--firedoc.netlify.com

File: source/apis/impressions/request-panel.mdx [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+    - Quick method selection
+    - Quick body type selection
+    - Add multiple bodies
+2. **Auths**
+    - Quick auth type selection
+3. **Headers**
+4. **Params**
+5. **Scripts**
+6. **Config**
+7. **Code snippets Generator**
```
Comment (Nishchit14) [inline source/apis/impressions/request-panel.mdx]: @vipulgupta2048 
Here 3-7 needs some brief description, do you have anything else in mind?

File: source/apis/impressions/request-panel.mdx [hunk 2]
```diff
@@ -1,13 +1,27 @@
...
+
+Requests can be used to retrieve, add, delete, and update data as well as can be sent with a wide array of request methods, parameters, authorization details, and anybody data that is needed.
```
Comment (Nishchit14) [inline source/apis/impressions/request-panel.mdx]: >  anybody data that is needed.

I didn't get this sentence. please share your views of it.

File: source/apis/impressions/request-panel.mdx (hunk 1 above)
Comment (vipulgupta2048) [reply source/apis/impressions/request-panel.mdx]: Well, everything will be linked to their own impression in the final deployment. We can do one-liners for each of them if you want?

Comment (Nishchit14) [reply source/apis/impressions/request-panel.mdx]: One linear is fine, and what if we can link them with their individual impression? just an idea.

**Comment by Nishchit14** [comment]
@vipulgupta2048 anything pending here?

File: source/apis/impressions/request-panel.mdx (hunk 1 above)
Comment (vipulgupta2048) [reply source/apis/impressions/request-panel.mdx]: Nishchit, that's what I said. Sure resolving.

File: source/apis/impressions/request-panel.mdx (hunk 2 above)
Comment (vipulgupta2048) [reply source/apis/impressions/request-panel.mdx]: Yeah, +1 on that. Removing this line. Even I don't feel the need for it that much.

**Comment by Nishchit14** [comment]
🚀 Deployed on https://5e7d40069da388992b69441a--firedoc.netlify.com

**Pull request merged**